mlir/test/auto_e2e/lit.cfg.py

```diff
@@ -0,0 +1,63 @@
+# -*- Python -*-
+
```

Comment:
@yiqian1 could you leave some documentation about how this `lit.cfg.py` is going to be picked up by the `check-e2e` target?

Reply:
I've added comments in the CMakeLists.txt.